db-checkbook/build_scripts/03_export.py

```diff
@@ -1,6 +1,57 @@
-from dcpy.connectors.s3 import client
+import os
+from datetime import date
+from pathlib import Path
+from dotenv import load_dotenv
+from dcpy.connectors import s3
 
-# TODO: upload data to DO 
+BASE_BUCKET = 'edm-publishing'
+_curr_file_path = Path(__file__).resolve()
+LIB_DIR = _curr_file_path.parent.parent / '.library'
+load_dotenv(_curr_file_path.parent.parent.parent / '.env') 
+
+
+def create_metadata_folder_structure(digital_ocean_file) -> dict:
+    today = date.today()
+    folder_structure = f'db-checkbook/main/{today}/{digital_ocean_file}'
+    dict_folder = {}
+    dict_folder['Folder Structure'] = folder_structure
+    return dict_folder
+
+def create_version_text_file() -> None:
+    today = date.today()
+    file_version = open(_curr_file_path.parent.parent / 'output'/ 'version.txt',"w+")
+    file_version.write(str(today))
+    return 
+
+def upload_final_file(final_file, digital_ocean_file) -> None:
+    """ final_file: str of output csv 
+        digital_ocean_file: str name of file in digital ocean to save  
+    """
+
+    s3_client = s3.client('s3', 
```

Comment:
Very minor note - this client doesn't need any args provided, these are the same as the defaults (and I'm surprised the positional `s3` didn't cause an error), so this can just be `s3.client()`. Once you push that I'll approve and we can get this merged!